omou-frontend/src/components/FeatureViews/Scheduler/Scheduler.js

```diff
@@ -291,123 +292,119 @@ class Scheduler extends Component {
-                                            inputProps={{
-                                                name: 'filterValue',
-                                                id: 'filter-class-type',
-                                            }}
-                                        >
-                                            <option value={"C"}>Class</option>
-                                            <option value={"T"}>Tutor</option>
-                                        </Select>
-                                    </FormControl>
-                                    :
-                                    <FormControl className={'filter-select'}>
-                                        <InputLabel htmlFor="filter-resource-type"></InputLabel>
-
-                                        <Select
-                                            native
-                                            value={this.state.resourceFilterValue}
-                                            onChange={this.handleResourceFilterChange('resourceFilterValue')}
-                                            inputProps={{
-                                                name: 'resourceFilterValue',
-                                                id: 'filter-resource-type',
-                                            }}
-                                        >
-                                            <option value={"R"}>Room</option>
-                                            <option value={"I"}>Instructors</option>
-                                        </Select>
-                                    </FormControl>
-                                }
+                <Paper className="paper" style={{ "padding": "2%" }}>
+                    <Typography variant="h3" align="left">Scheduler</Typography>
+                    <br />
+                    <Grid container
+                        direction="row"
+                        alignItems="center"
+                        className="scheduler-header">
+                        <Grid item className={"calendar-icon-header"}>
+                            <IconButton color={this.state.calendarIcon ? "primary" : ""} onClick={this.changeViewToCalendar} className={'calendar-icon'} aria-label='next-month'>
+                                <DateRangeOutlinedIcon />
+                            </IconButton>
+                        </Grid>
+                        <Grid item  >
+                            <IconButton color={this.state.resourceIcon ? "primary" : ""} onClick={this.changeViewToResource} className={'resource-icon'} aria-label='next-month'>
+                                <ViewListIcon />
+                            </IconButton>
+                        </Grid>
+                        <Grid item  >
+                            <IconButton onClick={""} className={'next-month'} aria-label='next-month'>
```

Comment:
`onClick={""}` should be removed or changed; it should always be a callback that can be executed, not a string - there is a warning thrown in the console.